Gemfile

```diff
@@ -49,7 +51,8 @@ end
 
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
-  gem "capybara"
-  gem "selenium-webdriver"
-  gem "webdrivers"
+  gem 'capybara'
+  gem 'factory_bot_rails'
```

Comment:
It's fine for this PR but, this is scope creep ideally you should avoid.

This PR is for Rubocop compliance, which has nothing to do with Factory Bot and the test suite. They're separate concerns in my opinion.

Keeping PRs as scoped to one concern is always worth doing.